src/components/Callout/Callout.tsx

```diff
@@ -22,11 +22,15 @@ export const Callout: React.FC<Props> = ({
     ...rest
 }) => {
     return (
-        <div {...rest} {...block({ context })}>
+        <div {...rest} {...block({ context, ...rest })}>
             <div {...elem('content', { context })}>{children}</div>
             {onRequestClose && (
-                <button {...elem('closeButton')} type="button" onClick={onRequestClose}>
-                    <MdClose {...elem('closeIcon')} />
+                <button
+                    {...elem('closeButton', { context })}
```

Comment:
why not adding rest here and in the previous one (line 27), but you do in the next one (MdClose, line 33)?